int8 nivel_arcondicionado
bool  radio
int32 servo_angulo
---
bool update_servo
